doc/apiv3-documentation.apib

```diff
@@ -1474,6 +1478,94 @@ This is an example of how a form might look like. Note that this endpoint does n
                 "message": "The resource you are about to edit was changed in the meantime."
             }
 
+# Group MyPreferences
+
+## Linked Properties
+|  Link     | Description                                              | Type          | Constraints           | Supported operations |
+|:---------:|----------------------------------------------------------| ------------- | --------------------- | -------------------- |
+| self      | This MyPreferences                                       | MyPreferences | not null              | READ                 |
+| user      | The current user that this preference belongs to         | User          | not null              | READ                 |
+
+## Local Properties
+| Property               | Description                                                | Type       | Constraints | Supported operations |
+|:----------------------:| -----------------------------------------------------------| ---------- | ----------- | -------------------- |
+| hideMail               | Hide mail address from other users                         | Boolean    |             | READ / WRITE         |
+| timeZone               | Current selected time zone                                 | String     |             | READ / WRITE         |
+| theme                  | Current selected theme                                     | String     |             | READ / WRITE         |
+| commentSortDescending  | Sort comments in descending order                          | Boolean    |             | READ / WRITE         |
+| warnOnLeavingUnsaved   | Issue warning when leaving a page with unsaved text        | Boolean    |             | READ / WRITE         |
+| accessibilityMode      | Enable accessibility mode                                  | Boolean    |             | READ / WRITE         |
+
+## MyPreferences [/api/v3/my_preferences]
+
++ Model
+    + Body
+
+            {
+                "_type" : "MyPreferences",
+                "_links" : {
+                    "self" : {
+                        "href" : "/api/v3/my_preferences",
+                    },
+                    "user": {
+                        "href": "/api/v3/users/1",
+                        "title": "John Sheppard"
+                    }
+                },
+                "hideMail" : false,
+                "timeZone" : "Berlin",
```

Comment:
This reminds me of the fact that timezones are hard.
Admittedly I did not have a look at standards yet, is "Berlin" a parsable timezone reference?

I see that "+0200" is probably not enough, as it does not indicate when and if I prefer to use DST and I see that timezones on a UI are referred to using big cities, still I am not sure whether "Berlin" is a good machine-readable identifier. What about "Amsterdam" and "Rathenow"?

I think I have to go out and read...

First read is [Wikipedia on tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), which would specify Identifiers like `Europe/Berlin` and `Europe/Amsterdam`.
This also seems to be the database/scheme used/promoted by IANA: http://www.iana.org/time-zones

I think those are quite important decisions regarding interoperability of an API. For instance this is why I promoted to use ISO8601 for times and durations, it is a common standard of formatting information.

I'd vote to use the format of the tz database, e.g. `Europe/Berlin`.

However, I am proposing this without any knowledge of the underlying data we use, so you may now disappoint me and say there is no way to determine those great strings ^^